src/pydash/helpers.py

```diff
@@ -142,9 +150,10 @@ def base_get(obj, key, default=NoValue):
     except Exception:
         pass
 
-    if not isinstance(obj, (dict, list)):
+    if not isinstance(obj, _attr_ignored_types) or hasattr(obj, '_fields'):
```

Comment:
For `namedtuple` detection, we can use `or (isinstance(obj, tuple) and hasattr(obj, "_fields"))` to help minimize false-positives since it's possible some other custom classes might define a `_fields` attribute but not be a namedtuple.

Reply:
Good catch!